airbyte-integrations/connectors/source-xero/source_xero/manifest.yaml

```diff
@@ -0,0 +1,2826 @@
+version: 0.78.5
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: CustomRecordExtractor
+      class_name: source_xero.components.CustomExtractor
+      field_path: ["{{ parameters.extractor_path }}"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.xero.com/api.xro/2.0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['access_token'] }}"
+    request_headers:
+      "Xero-Tenant-Id": "{{ config['tenant_id'] }}"
+      "Accept": "application/json"
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - type: HttpResponseFilter
+              action: IGNORE
+              predicate: >-
+                {{ 'App Only access is not allowed' in response
+                }}
+              http_codes:
+                - 403
+              error_message: Resource is forbidden, Please check all the scopes in xero platform for getting access of resource
+
+  paginator_base:
+    type: "DefaultPaginator"
+    page_size_option:
+      type: "RequestOption"
+      inject_into: "request_parameter"
+      field_name: "page_size"
+    pagination_strategy:
+      type: "PageIncrement"
+      page_size: 5
```

Comment:
Could you please set page to 100 as it was in previous version, or there are some reasons why we changing it?

Reply:
Done